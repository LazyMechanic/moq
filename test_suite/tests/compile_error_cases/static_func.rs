#[moq::automock]
trait Trait {
    fn f();
}

fn main() {}
