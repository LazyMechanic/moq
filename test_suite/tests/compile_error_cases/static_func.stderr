error: static functions not supported
 --> tests/compile_error_cases/static_func.rs:3:5
  |
3 |     fn f();
  |     ^^^^^^
